cryptotik/cryptopia.py

```diff
@@ -46,17 +46,22 @@ def __init__(self, apikey=None, secret=None, timeout=None, proxy=None):
 
         self.api_session = requests.Session()
 
+    def _verify_response(self, response):
+        '''verify if API responded properly and raise apropriate error.'''
+
+        assert response.status_code == 200, {'error: ' + str(response.json())}
```

Comment:
We have abandoned this for `result.raise_for_status()` in api or private_api methods.